### ISSUE TITLE
Добавить отображение количества выполненных задач в каждом блоке

### ISSUE THREAD

**Issue by SimonOsipov** [devmanorg/devman_v2#8]
Сейчас понять, сколько задач я сделал, сколько осталось в блоке можно понять только неявным способом (посмотреть на номер текущей задачи на проверке).

Предложение: добавить количество выполненных задач (например, 3/5) на карточку блока.
![image](https://user-images.githubusercontent.com/28685772/47092871-9f704b80-d230-11e8-9222-608f63d77bd7.png)

**Comment by SimonOsipov** [comment]
Жаль, не могу добавить Label к Issue
![image](https://user-images.githubusercontent.com/28685772/47092960-caf33600-d230-11e8-9873-0ffa0597b70b.png)

**Comment by vergeev** [comment]
@pelid мы можем разрешить создателям тикета выставлять лейблы?

**Comment by pelid** [comment]
Не можем. proof : https://stackoverflow.com/questions/13829466/how-to-put-a-label-on-an-issue-in-github-if-you-are-not-a-contributor-owner

Как выход из положения - предлагать лейбл в названии тикета, выглядеть будет так:

```
[enhancement] Отображение количества выполненных задач в каждом блоке
```

**Comment by vergeev** [comment]
@pelid нет, не надо, лучше самому ярлык вешать: что снаружи `enhancement`, для нас может быть багом или даже вонтфиксом.
